client/components/account-balances/strings.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const fundLabelStrings = {
+	available: __( 'Available funds', 'woocommerce-payments' ),
+	pending: __( 'Pending funds', 'woocommerce-payments' ),
+	reserve: __( 'Reserve funds', 'woocommerce-payments' ),
```

Comment:
Small typo: in the design, this is displayed as "Reserved".
```suggestion
	reserve: __( 'Reserved funds', 'woocommerce-payments' ),
```